pkg/apis/machine/v1beta1/machinehealthcheck_types.go

```diff
@@ -80,6 +80,15 @@ type MachineHealthCheckSpec struct {
 	// +kubebuilder:validation:Pattern="^([0-9]+(\\.[0-9]+)?(ns|us|µs|ms|s|m|h))+$"
 	// +kubebuilder:validation:Type:=string
 	NodeStartupTimeout metav1.Duration `json:"nodeStartupTimeout,omitempty"`
+
+	// MaxAge maximal age of a machine before it gets remediated
```

Comment:
Let's expand this to be a bit more about the use case
```suggestion
	// Machines older than this duration will be considered to have reached their maximal age and will be remediated.
	// This can be used to ensure that Machines are replaced frequently.
```